core/src/main/scala/filodb.core/binaryrecord2/RecordBuilder.scala

```diff
@@ -195,6 +249,8 @@ final class RecordBuilder(memFactory: MemFactory,
     recHash = combineHash(recHash, newHash)
   }
 
+  final def addFieldsFromBinRecord(base: Any, offset: Long, schema: RecordSchema): Unit = ???
```

Comment:
Adding a final method that always fails? Why?

Reply:
Forgot to remove that. Will do.